apps/web/src/scripts/test-auth-bypass.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
Is this a test of some kind? If it is, can we just have it as some test? Or should we delete it if this was just made for development?